g2o/benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+find_package(benchmark)
```

Comment:
Cool :)

Reply:
Should the benchmarks be put at the same level as unit_tests? At the moment they are in the same subdirectory as the library source code.

Reply:
IMHO it's better to keep in a separate sub folder.

Reply:
Do you mean I should leave as-is?

Reply:
as-is :+1: